### ISSUE TITLE
Banner checking

### ISSUE THREAD

**Issue by Huskydog9988** [zan8in/masscan#14]
From what I can tell, this library doesn't support [banner checking](https://github.com/robertdavidgraham/masscan#banner-checking) in masscan. How likely is this to get added?

**Comment by zan8in** [comment]
The "--banners" feature has been updated. Please try using it.

**Comment by Huskydog9988** [comment]
I'll check it out tomorrow maybe if I can find the time. Thank you for the quick turn around. Just to clarify, I need to manually parse for the banners in the output yea? (I did a quick review on my phone so idk if I missed something.)

**Comment by acidvegas** [comment]
yeah it doesnt show in the output at all lol....low effort repo tbh...
